organizations/views.py

```diff
@@ -20,23 +20,24 @@ def perform_create(self, serializer):
             user = User.objects.get(id=self.request.user.id)
         except User.DoesNotExist:
             return Response(status=status.HTTP_401_UNAUTHORIZED)
+                
+        if not user.terms_of_use:
+            # build this response
+            response = {
+                'status': 'error',
+                'code': status.HTTP_403_FORBIDDEN,
+                'message': 'Must agree to terms of use before proceeding',
+                'data': []
+            }
 
-        if not user.terms_of_use or not user.role:
```

Comment:
not sure why user.role is a requirement here?

Reply:
i think in the onboarding process we require users to agree to terms then state their role then create the org. this is probably better tbh since role probably isnt a hard requirement